### DISCUSSION TITLE
Welcome to Scroll Reverser Discussions!

### DISCUSSION THREAD

**Discussion by pilotmoon** [pilotmoon/Scroll-Reverser#100] [General]
## 👋 Welcome!
I'm giving this new Discussions feature a try. Feel free to post here. Don't be shy! -Nick

**Comment by qualiky** [comment]
Hey Nick! This is Sandeep. I'm from Nepal, and I really like Scroll Reverser! Been using it on my M1 MBP and it works flawlessly. I want to contribute and help on Nepali translation (if that's alright! 😄). If there's a way I can contribute, please let me know how. Thank you for your hard work and stay safe!

**Reply by pilotmoon** [reply]
HI Sandeep! I'd love to have a Nepali translation. There are two ways to do it.

1. Use the web-based localization platform: https://crwd.in/pilotmoon-apps
(You will need to create a free account to access translations)

2. Or alternatively if you are more technical, send me a Localizable.strings file, by pull request.

**Reply by pilotmoon** [reply]
Thanks for that - Nepali now in beta :) Update available in app.
![CleanShot 2021-02-05 at 11 40 52@2x](https://user-images.githubusercontent.com/17520/107029517-0c63d280-67a7-11eb-909e-bf3fda8b7078.png?s=100)

**Comment by qualiky** [comment]
Looks and works great, thank you for letting me contribute on it!
